### PR TITLE
Updates the SplashScreenLayout. Fixes #5 #6

### LINKED ISSUE #5: Splash Screen Layout needs to be updated
The Splash Screen Layout doesn't match with the mockups provided.

An update is needed to match the layout with mockups.

### PR DESCRIPTION
**Pull request by ShivamArora**
Updates the Splash Screen Layout as per the mockups.

Also fixes the issue of Splash Screen remaining open in background even after switching activity.

The layout now looks like this:
screenshot_2018-04-11-20-11-59-592_com udacity nanodegree blooddonation

Fixes #5 #6 

**What needs to be changed?**
The resource named ic_launcher with the app icon needs to be added to the project.

The splash screen will automatically reflect the changed app logo from the `ic_launcher` resource.

### CODE REVIEW

**Comment by ShivamArora** [comment]
@sakchhams Do you mean I need to put them in `dimens.xml`?

I designed the layout using the ConstraintLayout designer. Thus, resulting into hardcoded values.

**Comment by ShivamArora** [comment]
@sakchhams Take a look at the updates now.

Although there are some unnecessary file additions because of a merge conflict. Couldn't help that.

Someone in our project community suggested to separately commit these files too.

File: app/src/main/res/layout/activity_splash_screen.xml [hunk 1]
```diff
@@ -8,20 +8,52 @@
...
-        app:layout_constraintEnd_toEndOf="parent"
-        app:layout_constraintStart_toStartOf="parent"
-        app:layout_constraintTop_toTopOf="parent"
-    />
+  <TextView
+      android:id="@+id/tv_version_number"
+      android:layout_width="wrap_content"
+      android:layout_height="wrap_content"
+      android:layout_marginBottom="@dimen/dimen_60dp"
+      android:layout_marginEnd="@dimen/dimen_8dp"
```
Comment (sakchhams) [inline app/src/main/res/layout/activity_splash_screen.xml]: using names such as `dimen_8dp` defeats the purpose. Replace with something meaningful such as `splash_screen_tv_margin`

Comment (ShivamArora) [reply app/src/main/res/layout/activity_splash_screen.xml]: @sakchhams Updated the resource names. Is it fine now?

**Pull request closed without merging**